code/modules/mob/living/carbon/xenomorph/abilities.dm

```diff
@@ -39,6 +39,36 @@
 
 	return succeed_activate()
 
+// Toggle Bumpattacks
+/datum/action/xeno_action/bumpattack
+	name = "Toggle Bump Attacks"
+	action_icon_state = "bumpattack_off"
+	mechanics_text = "Automatically attack anyone you bump into."
+	var/active = FALSE
+
+/datum/action/xeno_action/bumpattack/action_activate()
+	var/mob/living/carbon/xenomorph/X = owner
+
+	add_cooldown()
```

Comment:
Does this have or should have a cooldown?

Reply:
oops, I was going to give it a cooldown then decided not to, forgot to remove that